src/bundle/Resources/public/css/structure.css

```diff
@@ -0,0 +1,64 @@
+html {
+    height: 100%;
+}
+
+body {
+    min-height: 100%;
+    display: flex;
+}
+
+ez-platform-ui-app {
+    display: flex;
+    flex: 1 0 100%;
+    flex-direction: column;
+}
+
+ez-navigation-hub {
+    flex: 0 0;
+}
+
+main {
+    flex: 1 0;
+    display: flex;
+}
+
+#discovery {
+    flex: 0 0 115px;
+    width: 115px;
+    background: var(--ez-color-discovery-bar);
+    color: var(--ez-color-text-white);
+}
+
+ez-toolbar .ez-button {
```

Comment:
No problem with that option :-) I treated them separately thinking on the extensibility cases these two bars have, highlighting they have to have a specific class given they are a particular case for buttons where they are grouped together and placed on both sides of the interface (plus the differences of style). But it's clear that from the code perspective they are what they are: buttons.

Ping us when you have more insights about the final option for icons :-)